bridge-history-api/internal/orm/batch_event.go

```diff
@@ -32,7 +32,7 @@ const (
 type BatchEvent struct {
 	db *gorm.DB `gorm:"column:-"`
 
-	ID               uint64     `json:"id" gorm:"column:id;primary_key"`
+	ID               uint64     `json:"id" gorm:"column:primary_key"`
```

Comment:
why change like this

Reply:
I thought it was duplicated, but yeah it's right to define a name for this column, so changed it back in [df944d1](https://github.com/scroll-tech/scroll/pull/1075/commits/df944d1137bad955520c4ae1178d9de7166e58af).